config/jobs/periodic/knative/eventing/eventing-main.yaml

```diff
@@ -0,0 +1,74 @@
+periodics:
+  - name: knative-eventing-main-periodic
+    labels:
+      preset-knative-powervs: "true"
+    decorate: true
+    cron: "0 3 * * *"
+    extra_refs:
+      - base_ref: main
+        org: ppc64le-cloud
+        repo: knative-upstream-ci
+        workdir: true
+      - base_ref: main
+        org: knative
+        repo: eventing
+    spec:
+      containers:
+        - image: quay.io/powercloud/knative-prow-tests:latest
+          resources:
+            requests:
+              cpu: "1500m"
+            limits:
+              cpu: "1500m"
+          command:
+            - runner.sh
+          args:
+            - bash
+            - -c
+            - |
+              set -o errexit
+              set -o nounset
+              set -o pipefail
+              set -o xtrace
+
+              TIMESTAMP=$(date +%Y%m%d%H%M%S%3N)
+              K8S_BUILD_VERSION=$(curl https://storage.googleapis.com/k8s-release-dev/ci/latest.txt)
+
+              cleanup() {
+                kubetest2 tf --powervs-region syd --powervs-zone syd05 \
```

Comment:
@valen-mascarenhas14 will similar change like https://github.com/ppc64le-cloud/test-infra/pull/506/files be required for these jobs too?
Also, @kabhiibm Is the ARTIFACTS path taken care for these jobs to have the JUnit tab displayed properly?